src/common/utils/date-time/format.ts

```diff
@@ -8,15 +8,17 @@ export const formatDateRange = ({
   startDate,
   endDate,
 }: {
-  startDate: string;
-  endDate: string;
+  startDate?: string;
+  endDate?: string;
 }): string => {
-  const isSingleDate: boolean = !endDate || startDate === endDate;
-  const formattedStartDate: string = formatDate(startDate);
-  if (isSingleDate) {
-    return formattedStartDate;
+  if (!startDate || (!startDate && !endDate)) {
```

Comment:
I noticed we don't yet have unit test for this function. It looks the kind of function that might be easy to unit test, so I wonder if it would be worth it?

Reply:
I'll add a test

Reply:
Added tests